scene/resources/font.cpp

```diff
@@ -498,6 +498,11 @@ Size2 Font::get_string_size(const String &p_string) const {
 void BitmapFont::set_fallback(const Ref<BitmapFont> &p_fallback) {
 
 	ERR_FAIL_COND(p_fallback == this);
+
+	for (Ref<BitmapFont> p_fallback_child = p_fallback; p_fallback_child != NULL; p_fallback_child = p_fallback_child->get_fallback()) {
```

Comment:
Two things:
- `p_` is for method parameters, not local variables, so it should be `fallback_child`.
- Might need an `ERR_EXPLAIN` to give a more descriptive error mentioning the recursion in the fallback chain.

Reply:
Bump.